diff -up ktuberling-20.03.90/toplevel.cpp.omv~ ktuberling-20.03.90/toplevel.cpp
--- ktuberling-20.03.90/toplevel.cpp.omv~	2020-04-14 00:30:19.515911670 +0200
+++ ktuberling-20.03.90/toplevel.cpp	2020-04-14 00:30:24.260964758 +0200
@@ -304,11 +304,7 @@ void TopLevel::setupKAction()
 
   PlaygroundDelegate *playgroundDelegate = new PlaygroundDelegate(playgroundCombo->view());
   playgroundCombo->setItemDelegate(playgroundDelegate);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
   connect(playgroundCombo, QOverload<int>::of(&KComboBox::currentIndexChanged), this, &TopLevel::changeGameboardFromCombo);
-#else
-  connect(playgroundCombo, QOverload<int, const QString &>::of(&KComboBox::currentIndexChanged), this, &TopLevel::changeGameboardFromCombo);
-#endif
   QWidgetAction *widgetAction = new QWidgetAction(this);
   widgetAction->setDefaultWidget(playgroundCombo);
   actionCollection()->addAction( QStringLiteral( "playgroundSelection" ),widgetAction);
